pkg/morph/client/container/client.go

```diff
@@ -90,26 +86,13 @@ type opts struct {
 	feePutNamedSet bool
 	feePutNamed    fixedn.Fixed8
 
-	disableNotarySigning bool
-
 	staticOpts []client.StaticClientOption
 }
 
 func defaultOpts() *opts {
 	return new(opts)
 }
 
-// DisableNotarySigning returns option to disable
-// notary request signing. With that option, every
-// call will be created, signed with provided key
-// (a single regular sign) and sent to the side
-// chain.
-func DisableNotarySigning() Option {
```

Comment:
This doesn't seem to be used, can be a separate patch.

Reply:
ok